source/lib/state_representation/include/state_representation/Robot/JointState.hpp

```diff
@@ -301,6 +316,13 @@ namespace StateRepresentation
 		this->velocities = velocities;
 	}
 
+	inline void JointState::set_velocities(const std::vector<double>& velocities)
```

Comment:
Here's an example where your local variable is shadowing the class member variable, e.g. overriding the symbol `velocities`. Your consistent use of `this->` makes this still readable so I think it's fine, but really shadowing member variables should be avoided where possible.

Reply:
Yes in state_representation it was not done yet but everywhere else class member variables are now underscored to resolve this issue. This standard should be enforced in state_representation as well